interface.js

```diff
@@ -0,0 +1,157 @@
+var session_key;
+var user_id;
+
+$(document).ready(function(){
+  var response;
+  var i;
+
+  $.get('https://chitter-backend-api.herokuapp.com/peeps', function(data) {
+    var response = data;
+    for (var i = 0; i < response.length; i++) {
+      var dateTime = response[i].created_at
+      peep = "<li id="+response[i].id+">"+ "<div class='peep-info'>@" + response[i].user.handle + " - " + getTimeValue(dateTime) +  
+            "</div> <div class='peep'>" + response[i].body + "</div></li>"
+      $('.peeps').append(peep);
+      individualPeep(response[i].id);
+    };
+  });
+  
+  function individualPeep(id) {
+    $('#'+id).click(function () {
+      popup();
+      $.get('https://chitter-backend-api.herokuapp.com/peeps/'+id, function(peep) {
+        $('.singlePeepInfo').text(peep.user.handle);
+        $('.singlePeep').text(peep.body);
+      });
+    });
+  };
+
+  $('.go').click(function() {
+    username = $('.username').val();
+    password = $('.password').val();
+    login(username, password);
+  });
+
+  $('.login').click(function () {
+    dropdown();
+  });
+
+  $('.logout').click(function() {
+    logout();
+  });
+
+  $('#post').click( function() {
+    var body = $('#text').val();
+    peep(body);
+  });
+
+  function logout() {
+    session_key = "";
+    user_id = "";
+    $('.logout').hide();
+    $('.login').show();
+    $('.new-peep').hide();
+  };
+
+  function login (username, password) {
+    $.ajax
+    ({
+        type: "POST",
+        url: 'https://chitter-backend-api.herokuapp.com/sessions',
+        contentType: 'application/json',
+        async: false,
+        data: JSON.stringify({"session": {"handle":username, "password":password}}),
+        success: function (data) {
+        successfulLogin(data); 
+      }
+    });
+  };
+
+  function peep(body) {
+    console.log('hello');
+    $.ajax
+    ({
+        type: "POST",
+        url: 'https://chitter-backend-api.herokuapp.com/peeps',
+        contentType: 'application/json',
+        Authorization: 'Token token=' + session_key,
+        async: false,
+        data: JSON.stringify({"peep": {"user_id":user_id, "body":body}}),
+        success: function (data) {
+        successfulLogin(data); 
+      }
+    });
+  };
+
+  function successfulLogin(data) {
+    session_key = data.session_key;
+    user_id = data.user_id;
+    $('.login').hide();
+    $('.logout').show();
+    $('.new-peep').show();
+    $(".logindropdown").fadeOut(); 
+  };
+
+  function popup() {
+    $('.popup').fadeIn(500);
+    $(".cover").fadeTo(500, 0.5);
+
+    $(".close").click(function () {
+      $(".popup").fadeOut(500);
+    });
+  };
+
+  function dropdown() {
+    $('.logindropdown').slideDown();
+
+    $(".close").click(function () {
+      $(".logindropdown").slideUp();
+    });
+  };
+
+  function getTimeValue(datetime) {
```

Comment:
Love what you have done with relative time! This could perhaps be simplified by using a case statement and creating sub-functions